File: WebBiblioteca_Api/BibliotecaApiDLL/Emprestimo/Emprestimo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BibliotecaApiDLL.Emprestimo
{
    public class Emprestimo
    {
        public Emprestimo(int codEmprestimo, string leitor, string cpfleitor, string livro, int exemplar, string dataEmprestimo, string dataDevolucao, string status)
        {
            CodEmprestimo = codEmprestimo;
            Leitor = leitor;
            Cpfleitor = cpfleitor;
            Livro = livro;
            Exemplar = exemplar;
            DataEmprestimo = dataEmprestimo;
            DataDevolucao = dataDevolucao;
            Status = status;
        }

        public int CodEmprestimo { get; set; }
        public string Leitor { get; set; }
        public string Cpfleitor { get; set; }
        public string Livro { get; set; }
        public int Exemplar { get; set; }
        public string DataEmprestimo { get; set; }
        public string DataDevolucao { get; set; }
        public string Status { get; set; }
    }
}

File: WebBibliotecaApp/src/Objects/emprestimo.ts
export class Emprestimo {
  codEmprestimo = 0;
  leitor: string = "";
  cpfleitor: string = "";
  livro: string = "";
  exemplar = 0;
  dataEmprestimo: string = "";
  dataDevolucao: string = "";
  status: string = "";

  constructor(codEmprestimo: number, leitor: string, cpfleitor: string, livro: string, exemplar: number, dataEmprestimo: string, dataDevolucao: string, status: string){
    this.codEmprestimo = codEmprestimo;
    this.leitor = leitor;
    this.cpfleitor = cpfleitor;
    this.livro = livro;
    this.exemplar = exemplar;
    this.dataEmprestimo = dataEmprestimo;
    this.dataDevolucao = dataDevolucao;
    this.status = status;
  }
}
